resources/translations/en.edn

```diff
@@ -256,7 +256,7 @@
                       :invalid-value "Invalid value."
                       :required "Field \"%1\" is required."
                       :toolong "Answer to field \"%1\" is too long."}}
-  :header {:title "Welcome to REMS"}
+  :header {:title "REMS"}
```

Comment:
A nice change, brevity is good here (not to mention that "Welcome to" stuck in the page title as long as you kept using the site, making it overstay its welcome at some point...)